react/features/base/dialog/reducer.js

```diff
@@ -0,0 +1,33 @@
+import {
+    BEGIN_DIALOG_REQUEST,
+    END_DIALOG_REQUEST
+} from './actionTypes';
+
+import { ReducerRegistry, setStateProperties } from '../redux';
```

Comment:
The imports from the other modules should be ordered before the imports from the same module.